static/docs/commands-reference/pull.md

```diff
@@ -1,53 +1,123 @@
 # pull
 
-Pulls data files to the working space.
+Pulls data files to the local workspace from a remote workspace.
 
-The set of data files to pull (usually it means downloading from the remote
-storage if file is not in the local cache yet) is determined by analyzing all
-`.dvc` files in the current branch, unless `--all-branches` is specified.
+## Usage
 
-The command pulls only output of a specific stage if dvc file is specified `dvc pull data.zip.dvc`.
+```usage
+    usage: dvc pull [-h] [-q | -v] [-j JOBS]
+                    [--show-checksums] [-r REMOTE]
+                    [-a] [-T] [-d] [-f] [-R]
+                    [targets [targets ...]]
+
+    Pull data files from the cloud.
+
+    positional arguments:
+      targets               DVC files.
+```
+
+## Description
+
+DVC supports sharing a data cache between workspaces.  Members of a team may
+be working with the same data, but using different algorithms, and will find
+it useful to share data.  The `dvc pull` command is analogous to `git pull`,
+and allows one to retrieve data from a remote cache.
```

Comment:
Note, even here you use a different term - remote cache not remote workspace.

Reply:
DVC pull and DVC push both work with remote cache, so shouldn't I use that phrase throughout?

Reply:
I would use just remote or remote storage to not mix it with local cache with a link to the dvc remote page.

Reply:
Reworded this section